### PR TITLE
FIX parameters array_merge + global activation

### PR DESCRIPTION
**Pull request by inoveaconseil**

### CODE REVIEW

**Comment by eldy** [comment]
The PR introduce the new feature CaRE_GRANTCHILD
Can you describe the goal of thisnoption and push it into develop instead if 13.0...

**Comment by inoveaconseil** [comment]
The code was incorrect and with this parameter, we can manage the argument. If you want manage the rank of all the child.

File: htdocs/core/class/commonobject.class.php [hunk 1]
```diff
@@ -2671,7 +2673,7 @@ public function getChildrenOfLine($id, $includealltree = 0)
 			if ($this->db->num_rows($resql) > 0) {
 				while ($row = $this->db->fetch_row($resql)) {
 					$rows[] = $row[0];
-					if (!empty($includealltree)) $rows = array_merge($rows, $this->getChildrenOfLine($row[0]), $includealltree);
+					if (!empty($includealltree)) $rows = array_merge($rows, $this->getChildrenOfLine($row[0]));
```
Comment (eldy) [inline htdocs/core/class/commonobject.class.php:2676]: Why removing the feature $includealltree ?

**Comment by eldy** [comment]
It seems this PR was set with "Discussion" or "PR to fix" tag, but no answer nor correction was provided since a long time. So request is automatically closed.
Please reopened and add a comment if you think this is an error.

**Pull request closed without merging**